api/host_query_db.py

```diff
@@ -566,23 +567,18 @@ def get_host_ids_list(
 
 
 def get_hosts_to_export(
-    identity: object,
-    filters: object = {},
-    export_format: str = "json",
-    rbac_filter: dict = {},
-) -> list:
+    identity: object, filters: object = {}, export_format: str = "json", rbac_filter: dict = {}, batch_size: int = 0
+):
```

Comment:
```suggestion
) -> Iterator[dict]:
```

Note that you'll need to add this to your imports:
`from typing import Iterator`

Reply:
Added this suggestion!